jekyll-assets/_layouts/boxes.html

```diff
@@ -34,6 +34,31 @@ <h1><a href="/documentation/"><b>Raspberry Pi</b> Documentation</a></h1>
       </section>
     </div>
 
+    <div id="related" role="complementary">
+      <div id="related-title">
+        <h2>Related resources</h2>
+      </div>
+      <div id="related-container" role="complementary">
+      <section id="related-content">
+            <a class="box" href="https://pip.raspberrypi.com/">
+              <span><img src="/documentation/images/full-sized/PIP.png" width="121" height="121" alt=""></span>
+              <span class="title">Product Information Portal</span>
+              <span>Compliance documentation</span>
```

Comment:
I believe that PIP was _originally_ just "Compliance documentation", but it's much more than that now? Product Briefs, Whitepapers, Datasheets, etc.